Singularity/Singularity/Screen/IWindowItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Singularity.Property;
+
+namespace Singularity.Screen
+{
+    interface IWindowItem: IDraw, IUpdate
```

Comment:
is that not already included in the other PR? see my comment there

see #126

Reply:
Accidentally edited in both branches i think